### PR TITLE
Spelling locals

### PR DESCRIPTION
**Pull request by jsoref**
These changes are mostly to arguments, local variables, local functions, and non-public methods (mostly private, some protected), and a couple of comments (which I apparently failed to group w/ a previous comments only PR).
I'm trying to exclude public methods from this series (that's reserved for another PR later) -- If there are public methods in here, I'll gladly drop them.

As there are quite a few changes, I've tried to group changes into commits per directory tree.

### CODE REVIEW

**Comment by vors** [comment]
There are compilation errors introduced by your changes (click red crosses to go to the detailed logs). Please fix them first, before we evaluate the PR

**Comment by jsoref** [comment]
I was worried `event` might be a reserved word, but I foolishly checked the PowerShell list instead of the C# list. The canonical spelling appears to be `evt` instead of `evnt`, so I've selected that.

The second failure is apparently a selection miss (I'm basically manually recording a subset of changes into commits -- and apparently I missed one).

**Comment by jsoref** [comment]
@vors sorry about that, I've re-pushed a version which doesn't upset the builders.

File: src/Microsoft.Management.Infrastructure.CimCmdlets/Utils.cs [hunk 1]
```diff
@@ -203,7 +203,7 @@ internal static bool GenrateVerboseMessage
         internal static string removeParameterSetName = @"Remove parameter set {0} name from cache";
         internal static string currentParameterSetNameCount = @"Cache have {0} parameter set names";
         internal static string currentParameterSetNameInCache = @"Cache have parameter set {0} valid {1}";
-        internal static string currentnonMadatoryParameterSetInCache = @"Cache have optional parameter set {0} valid {1}";
+        internal static string currentnonMandatoryParameterSetInCache = @"Cache have optional parameter set {0} valid {1}";
```
Comment (vors) [inline src/Microsoft.Management.Infrastructure.CimCmdlets/Utils.cs]: I don't see any usages of any of these internal strings. @lzybkr We should investigate and consider removing them.

File: src/Microsoft.PowerShell.Commands.Management/commands/management/Service.cs [hunk 2]
```diff
@@ -683,9 +683,9 @@ protected override void ProcessRecord()
                     }
                     if (RequiredServices.IsPresent)
                     {
-                        foreach (ServiceController servdependedon in service.ServicesDependedOn)
+                        foreach (ServiceController servicedependedon in service.ServicesDependedOn)
```
Comment (vors) [inline src/Microsoft.PowerShell.Commands.Management/commands/management/Service.cs]: Such minor changes may be not necessary, but thank you for fixing it as well.

File: src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs [hunk 3]
```diff
@@ -2851,10 +2851,10 @@ private static bool IsAvailableFarEastCodePage(uint codePage)
         private static bool IsAnyDBCSCharSet(uint charSet)
         {
             const uint SHIFTJIS_CHARSET = 128;
-            const uint HANGEUL_CHARSET = 129;
+            const uint HANGUL_CHARSET = 129;
```
Comment (vors) [inline src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs]: Both spellings are correct 

> HANGUL_CHARSET:  Also spelled "Hangeul". Specifies the Hangul Korean character set.

https://msdn.microsoft.com/en-us/library/cc250412.aspx

File: src/Microsoft.PowerShell.PSReadLine/BasicEditing.cs [hunk 4]
```diff
@@ -428,8 +428,8 @@ private bool UnresolvedCommandCouldSucceed(string commandName, Ast rootAst)
                 {
                 // The following are debugger commands that should be accepted if we're debugging
                 // because the console host will interpret these commands directly.
-                case 's': case 'v': case 'o': case 'c': case 'q': case'k': case 'l':
-                case 'S': case 'V': case 'O': case 'C': case 'Q': case'K': case 'L':
+                case 's': case 'v': case 'o': case 'c': case 'q': case 'k': case 'l':
+                case 'S': case 'V': case 'O': case 'C': case 'Q': case 'K': case 'L':
```
Comment (vors) [inline src/Microsoft.PowerShell.PSReadLine/BasicEditing.cs]: These changes would be better as a separate PR

File: src/Microsoft.PowerShell.PackageManagement/Utility/HostApiExtensions.cs [hunk 5]
```diff
@@ -82,7 +82,7 @@ internal static class HostApiExtensions {
         }
 
         [SuppressMessage("Microsoft.Performance", "CA1811:AvoidUncalledPrivateCode", Justification = "Required.")]
-        internal static IHostApi SuprressBootstrapping(this IHostApi parent, bool isProcessing) {
+        internal static IHostApi SuppressBootstrapping(this IHostApi parent, bool isProcessing) {
```
Comment (vors) [inline src/Microsoft.PowerShell.PackageManagement/Utility/HostApiExtensions.cs]: @lzybkr this API is not used anywhere

File: src/Microsoft.WSMan.Management/CredSSP.cs [hunk 6]
```diff
@@ -440,7 +440,7 @@ public class EnableWSManCredSSPCommand : WSManCredSSPCommandBase, IDisposable/*,
 
         //private const string DelegateComputerParam = "DelegateComputer";
         //private String[] delegatecomputer;
-        //private RuntimeDefinedParameterDictionary dyanmicParameters = new RuntimeDefinedParameterDictionary();
+        //private RuntimeDefinedParameterDictionary dynamicParameters = new RuntimeDefinedParameterDictionary();
```
Comment (vors) [inline src/Microsoft.WSMan.Management/CredSSP.cs]: This code is commented out. I verified that all usages were replaced. @lzybkr do we want to remove this unused code?

File: src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs (hunk 3 above)
Comment (jsoref) [inline src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs]: I can drop this change if you like.

As someone who has spent over a decade working on Web Browsers, a significant amount of time dealing w/ character sets, Hangeul is definitely not the primary spelling (13million vs 1/2 million). It's a transliteration, so it isn't uncommon for there to be multiple transliterations.

Notably, the url you provided as a reference uses the `e`-less version in its enum.

File: src/Microsoft.PowerShell.PSReadLine/BasicEditing.cs (hunk 4 above)
Comment (jsoref) [inline src/Microsoft.PowerShell.PSReadLine/BasicEditing.cs]: Will do shortly.

File: src/Modules/Shared/Microsoft.PowerShell.Archive/Microsoft.PowerShell.Archive.psm1 [hunk 7]
```diff
@@ -93,7 +93,7 @@ function Compress-Archive
             $destinationParentDir = '.'
         }
 
-        $achiveFileName = [system.IO.Path]::GetFileName($DestinationPath)
+        $archiveFileName = [system.IO.Path]::GetFileName($DestinationPath)
```
Comment (vors) [inline src/Modules/Shared/Microsoft.PowerShell.Archive/Microsoft.PowerShell.Archive.psm1]: Lets not touch Archive module for a moment. We have a separate repo for it https://github.com/PowerShell/Microsoft.PowerShell.Archive and will probably replace archive code in this repo by a git submodule or something similar.

File: src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs (hunk 3 above)
Comment (jsoref) [inline src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs]: Presumably `Hangule` (also fixed in this series) isn't accepted. I'll split this change out as well since it's adjacent to the `case`-`space` change.

Comment (vors) [inline src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs]: No, I think we should use `HANGUL_CHARSET` as you suggest in the PR.
I was curious and researched the subject a little bit and just shared my findings.

Comment (jsoref) [inline src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs]: FWIW, I generally try to change things to have a single consistent spelling. If there was a dominant spelling in the project, e.g. an en-GB spelling of `colour`, then I would have settled on that. Currently there are three spellings used about evenly. Given that, my default was to pick the single most commonly used spelling (in this case, a word with which I'm familiar, although I will also ask Google to confirm prevalence).

`Hangul`: https://github.com/PowerShell/PowerShell/search?utf8=%E2%9C%93&q=hangul
`Hangeul`: https://github.com/PowerShell/PowerShell/search?utf8=%E2%9C%93&q=HANGEUL_CHARSET&type=Code
`Hangule`: https://github.com/PowerShell/PowerShell/search?utf8=%E2%9C%93&q=hangule&type=Code

File: src/Modules/Shared/PowerShellGet/PSModule.psm1 [hunk 8]
```diff
@@ -122,7 +122,7 @@ $script:NuGetProviderName = "NuGet"
 $script:NuGetProviderVersion  = [Version]'2.8.5.201'
 
 $script:SupportsPSModulesFeatureName="supports-powershell-modules"
-$script:FastPackRefHastable = @{}
+$script:FastPackRefHashtable = @{}
```
Comment (vors) [inline src/Modules/Shared/PowerShellGet/PSModule.psm1]: Can we separate script files (.psm1) into a separate PR?
C# code has static compile checks, but psm1 files would need more attention.

File: src/System.Management.Automation/CoreCLR/CorePsStub.cs [hunk 9]
```diff
@@ -1207,20 +1207,20 @@ public RegistryValueKind GetValueKind(string name)
 
         public void Close()
         {
-            throw new NotImplementedException("Close() is not implemented. TransactedRegistry related APIs should not be hitten.");
+            throw new NotImplementedException("Close() is not implemented. TransactedRegistry related APIs should not be hidden.");
```
Comment (vors) [inline src/System.Management.Automation/CoreCLR/CorePsStub.cs]: hitten is correct here. It means, this code path should not be hit (executed).

File: src/Modules/Shared/Microsoft.PowerShell.Archive/Microsoft.PowerShell.Archive.psm1 (hunk 7 above)
Comment (jsoref) [inline src/Modules/Shared/Microsoft.PowerShell.Archive/Microsoft.PowerShell.Archive.psm1]: Sure

File: src/Modules/Shared/PowerShellGet/PSModule.psm1 (hunk 8 above)
Comment (jsoref) [inline src/Modules/Shared/PowerShellGet/PSModule.psm1]: OK

File: src/System.Management.Automation/CoreCLR/CorePsStub.cs (hunk 9 above)
Comment (jsoref) [inline src/System.Management.Automation/CoreCLR/CorePsStub.cs]: Normally that would be spelled `hit`. Or one could use `run` or `executed` for clarity.

File: src/System.Management.Automation/engine/remoting/client/Job2.cs [hunk 10]
```diff
@@ -873,9 +873,9 @@ public override void StartJob()
                 // Check to see expected behavior if one child job fails to start.
             } 
 
-            if (ExcecutionError.Count == 1)
+            if (ExecutionError.Count == 1)
             {
-                throw ExcecutionError[0];
+                throw ExecutionError[0];
```
Comment (vors) [inline src/System.Management.Automation/engine/remoting/client/Job2.cs]: @lzybkr should we remove this commented code?

File: src/System.Management.Automation/engine/remoting/commands/CustomShellCommands.cs [hunk 11]
```diff
@@ -2952,8 +2952,8 @@ public sealed class SetPSSessionConfigurationCommand : PSSessionConfigurationCom
 ";
 
         private const string setSessionConfigurationTimeoutQuotasSbFormat = @"
-function Set-SessionPluginIdleTimeoutQuotas([int] $maxIdleTimeoutms, [int] $idleTimeoutms, [bool] $setMaxIdleTimoutFirst) {{
-    if ($setMaxIdleTimoutFirst) {{
+function Set-SessionPluginIdleTimeoutQuotas([int] $maxIdleTimeoutms, [int] $idleTimeoutms, [bool] $setMaxIdleTimeoutFirst) {{
```
Comment (vors) [inline src/System.Management.Automation/engine/remoting/commands/CustomShellCommands.cs]: oh, these C# strings that represents powershell are the worst for editing :)

This `$setMaxIdleTimeoutFirst` change looks good.

File: src/System.Management.Automation/engine/remoting/common/remotingexceptions.cs [hunk 12]
```diff
@@ -67,7 +67,7 @@ internal enum PSRemotingErrorId : uint
         MissingIsStartFragment = 409,
         MissingProperty = 410,
         ObjectIdsNotMatching = 411,
-        FragmetIdsNotInSequence = 412,
```
Comment (vors) [inline src/System.Management.Automation/engine/remoting/common/remotingexceptions.cs]: `PSRemotingErrorId` is tightly coupled with `RemotingErrorIdStrings`. We need to change them together.

Comment (jsoref) [inline src/System.Management.Automation/engine/remoting/common/remotingexceptions.cs]: Oops, sorry. I tried to split such things out, I guess I missed this correlation.
I have a non public commit:
`spelling: public identifiers in src/System.Management.Automation`
which is where I'll send this.

File: src/System.Management.Automation/CoreCLR/CorePsStub.cs (hunk 9 above)
Comment (jsoref) [inline src/System.Management.Automation/CoreCLR/CorePsStub.cs]: ... or `called`.

@vors if you suggest a replacement (`called`, `executed`, `hit`, `run`, or `used`), I can fix it up (or I can split it off into its own PR if that helps this one merge faster).

File: src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs (hunk 3 above)
Comment (vors) [inline src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs]: As I said previously, let's do this change to `HANGUL_CHARSET`
I just posted my findings but didn't want you to remove them from the PR.

Comment (vors) [inline src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs]: Oh you send another PR I see :)

File: src/System.Management.Automation/CoreCLR/CorePsStub.cs (hunk 9 above)
Comment (vors) [inline src/System.Management.Automation/CoreCLR/CorePsStub.cs]: Yes, the separate PR is always better :)

File: src/Microsoft.PowerShell.PSReadLine/Completion.cs [hunk 13]
```diff
@@ -356,8 +356,8 @@ private void PossibleCompletionsImpl(CommandCompletion completions, bool menuSel
             ConsoleBufferBuilder cb;
             if (Options.ShowToolTips)
             {
-                const string seperator = "- ";
```
Comment (vors) [inline src/Microsoft.PowerShell.PSReadLine/Completion.cs]: PSReadLine is another module that has its own repo https://github.com/lzybkr/PSReadLine and will be replaced by git submodule or something similar

**Comment by vors** [comment]
Overall looks good!

Before we can merge, please

- [ ] don't apply PSReadLine fixes
- [ ] separate `hitted` replacement in a separate PR

BTW, extra thank you for keeping the history clean!

**Comment by jsoref** [comment]
@vors: done

File: src/Microsoft.PowerShell.PackageManagement/Utility/HostApiExtensions.cs (hunk 5 above)
Comment (daviwil) [inline src/Microsoft.PowerShell.PackageManagement/Utility/HostApiExtensions.cs]: I don't see it anywhere in the ISE codebase

**Comment by vors** [comment]
Thank you @jsoref !
